src/widget/wbattery.h

```diff
@@ -16,8 +16,8 @@ class Battery;
 class WBattery : public WWidget {
     Q_OBJECT
   public:
-    WBattery(QWidget* parent=NULL);
-    virtual ~WBattery();
+    explicit WBattery(QWidget* parent=nullptr);
```

Comment:
I think this tidy pass is not redoing all of our whitespace at all, that would be a different and much bigger diff.  It's something we should definitely try to do, though!